### PR TITLE
feat: 1723 wcag landing dashboard

### LINKED ISSUE #1723: WCAG standards landing dashboard
**As a** *SPAR User* **I want** *SPAR to follow WCAG standards* **So That** *I can use assistive technologies to help me complete my work*

**Acceptance Criteria**

For the SPAR landing dashboard
image

- [x] Semantic HTML structure
- [x] Ensure all images have alternative text (for the photo on the log in screen use: "Picture of spruce tree branches with needles and cones".
- [x] Accessible forms
- [ ] Keyboard navigation
- [x] ARIA (Accessible Rich Internet Applications) roles and attributes
- [ ] Focus management
- [x] Contrast above 4.5:1
- [x] Tested - Use accessibility tools and screen readers during development.
   
**Additional Context**
- More information here: https://github.com/bcgov/nr-spar/wiki/Coding-for-accessibility:-Screen-readers

**Definition of Done**
- [x] Ready to Demo in Sprint Review
- [x] Does what I have made have appropriate test coverage?
- [ ] Documentation and/or scientific documentation exists and can be found
- [x] Peer Reviewed by 2 people on the team
- [ ] Manual testing of all PRs in Dev and Prod
- [x] QA from UX team member
- [ ] Merged

### PR DESCRIPTION
**Pull request by ngunner15**
# Description
Closes #1723 

#### New
- WCAG changes to the Dashboard page

#### Changed
- Dashboard page

#### Removed
- None

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests
- [x] Used [Wave](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) chrome extension

###  What gif/image best describes this PR or how it makes you feel?

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-1-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1751-backend.apps.silver.devops.gov.bc.ca/swagger-ui/index.html)
- [Oracle-API](https://nr-spar-1751-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

**Review by xiaopeng0202 — APPROVED** [review]

File: frontend/src/types/FavActivityTypes.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ export type FavActivityType = {
   header: string;
   link: string;
   highlighted: boolean;
+  role?: string;
```
Comment (mgaseta) [inline frontend/src/types/FavActivityTypes.ts:8]: Why do we need this field? I don't see it being used anywhere...

**Review by mgaseta — APPROVED** [review]
Just one small question, other than that LGTM!

File: frontend/src/types/FavActivityTypes.ts (hunk 1 above)
Comment (ngunner15) [reply frontend/src/types/FavActivityTypes.ts:8]: I added this because I wanted to put role="heading" to FavoriteCards, its not necessary, I removed it.

Comment (ngunner15) [reply frontend/src/types/FavActivityTypes.ts]: https://github.com/bcgov/nr-spar/pull/1751/commits/8fe6ec2770987c36ed49bb7950a874e04ffc8b8c

**Review by mgaseta — APPROVED** [review]

**Pull request merged**